src/message/MessageFull.js

```diff
@@ -75,20 +74,20 @@ class MessageFull extends PureComponent {
           />
           <View style={styles.contentWrapper}>
             <ScrollView style={styles.inner}>
-              <TouchableWithoutFeedback onLongPress={onLongPress}>
+              <TouchableOpacity onLongPress={onLongPress}>
```

Comment:
I think TouchableOpacity looks better when onLongPress, hence I used this and extracted this to another commit, if you don't agree push without this commit!